test/unit/range/views/view_minimiser_test.cpp

```diff
@@ -0,0 +1,137 @@
+                                                std::list<seqan3::dna4> const,
+                                                std::forward_list<seqan3::dna4>,
+                                                std::forward_list<seqan3::dna4> const>;
+
+TYPED_TEST_SUITE(minimiser_view_properties_test, underlying_range_types, );
+
+class minimiser_test : public ::testing::Test
+{
+protected:
+    std::vector<seqan3::dna4> text1{"AAAAAAAAAA"_dna4};
+    std::vector<seqan3::dna4> text1_short{"AAAAAA"_dna4};
+    result_t result1{0, 0, 0}; // Same result for ungapped and gapped
+    result_t result1_short{0}; // windows_size == text_size, same result for ungapped and gapped
+
+    std::vector<seqan3::dna4> text2{"AC"_dna4};
+    result_t result2{}; // Same result for ungapped and gapped
+
+    seqan3::bitcompressed_vector<seqan3::dna4> text3{"ACGGCGACGTTTAG"_dna4};
+    result_t result3_ungapped_no_rev{26, 97, 27}; // ACGG, CGAC, ACGT
+    result_t result3_gapped_no_rev{2, 5, 3};      // A--G, C--C-, A--T "-" for gap
+    result_t result3_ungapped_no_rev_stop{26, 97};// For stop at first T
+    result_t result3_gapped_no_rev_stop{2, 5};    // For stop at first T
+};
+
+TYPED_TEST(minimiser_view_properties_test, concepts)
+{
+    TypeParam text{'A'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4, 'C'_dna4, 'G'_dna4, 'A'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4,
+                'T'_dna4, 'T'_dna4, 'A'_dna4, 'G'_dna4}; // ACGTCGACGTTTAG
+    auto v = text | kmer_view | minimiser_view;
+    EXPECT_TRUE(std::ranges::input_range<decltype(v)>);
+    EXPECT_TRUE(std::ranges::forward_range<decltype(v)>);
+    EXPECT_FALSE(std::ranges::bidirectional_range<decltype(v)>);
+    EXPECT_FALSE(std::ranges::random_access_range<decltype(v)>);
+    EXPECT_TRUE(std::ranges::view<decltype(v)>);
+    EXPECT_FALSE(std::ranges::sized_range<decltype(v)>);
+    EXPECT_FALSE(std::ranges::common_range<decltype(v)>);
+    EXPECT_TRUE(seqan3::const_iterable_range<decltype(v)>);
+    EXPECT_FALSE((std::ranges::output_range<decltype(v), size_t>));
+}
+
+TYPED_TEST(minimiser_view_properties_test, different_inputs_kmer_hash)
+{
+    TypeParam text{'A'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4, 'C'_dna4, 'G'_dna4, 'A'_dna4, 'C'_dna4, 'G'_dna4, 'T'_dna4,
+                'T'_dna4, 'T'_dna4, 'A'_dna4, 'G'_dna4}; // ACGTCGACGTTTAG
+    result_t ungapped_no_rev{27, 97, 27};                // ACGT, CGAC, ACGT
+    result_t gapped_no_rev{3, 5, 3};                     // A--T, C--C, A--T - "-" for gap
+    EXPECT_EQ(ungapped_no_rev, text | kmer_view | minimiser_view | seqan3::views::to<result_t>);
+    EXPECT_EQ(gapped_no_rev, text | gapped_kmer_view | minimiser_view | seqan3::views::to<result_t>);
+}
+
+TEST_F(minimiser_test, ungapped_kmer_hash)
+{
+    EXPECT_EQ(result1, text1 | kmer_view | minimiser_view | seqan3::views::to<result_t>);
+    EXPECT_THROW(text1_short | kmer_view | minimiser_view2, std::invalid_argument);
+    EXPECT_EQ(result2, text2 | kmer_view | seqan3::views::to<result_t>);
```

Comment:
1. why is there no minimizer view here?

2. please change `text2` to `too_short_text`
 
3. change the following
```suggestion
    auto empty_view = too_short_text | kmer_view;
    EXPECT_TRUE(std::ranges::empty(empty_view));
```
It will get rid of the error on jenkins.

Change the same thing in the gapped test